content/intro-to-storybook/angular/es/conclusion.md

```diff
@@ -7,19 +7,19 @@ description: 'Pon todo tu conocimiento junto y aprende más técnicas de Storybo
 
 [📕 **Repositorio de GitHub: chromaui/learnstorybook-code**](https://github.com/chromaui/learnstorybook-code)
 <br/>
-[🌎 **Storybook Desplegado**](https://clever-banach-415c03.netlify.com/)
+[🌎 **Storybook Desplegado**](https://clever-banach-415c03.netlify.app/)
 
 Storybook es una poderosa herramienta para React, Vue y Angular. Cuenta con una próspera comunidad de desarrolladores y una gran cantidad de complementos. Esta introducción demuestra una pequeña parte de todo lo que es posible con Storybook. Estamos seguros de que una vez que adoptes Storybook, quedarás impresionado por lo productivo que es crear interfaces de usuario duraderas.
 
 ## Aprende más
 
 ¿Quieres profundizar tus conocimientos? Aquí encontrarás algunos recursos útiles:
 
-- [**Documentación oficial de Storybook**](https://storybook.js.org/basics/introduction/) contiene la documentación del API, links comunitarios y una galería de complementos.
+- [**Documentación oficial de Storybook**](https://storybook.js.org/docs/react/get-started/introduction) contiene la documentación del API, links comunitarios y una galería de complementos.
```

Comment:
@gregrickaby when can please update from "/docs/react/get-started/introduction" to "/docs/angular/get-started/introduction" in the "Official Storybook documentation" link.